### ISSUE TITLE
Routes are not being used/get overwritten

### ISSUE THREAD

**Issue by ice8lue** [divshot/superstatic#147]
We're using a configuration like this for our app:

```javascript
routes: {
    "/admin/**": "/admin/index.html",
    "**": "index.html"
}
````

This was working fine before, but for some days now (new builds) the first rule is not working anymore (maybe gets overwritten by the second one). Requests like "/admin/dashboard" are routed to index.html rather than admin/index.html.

**Comment by scottcorgan** [comment]
@ice8lue what does your full divshot.json file look like?

**Comment by ice8lue** [comment]
```javascript
{
  "name": "our-app-id",
  "root": "./dist/client/web/",
  "clean_urls": false,
  "error_page": "error.html",
  "routes": {
    "/eyoadmin/**": "/eyoadmin/index.html",
    "/admin/**": "/admin/index.html",
    "**": "index.html"
  }
}
```

**Comment by scottcorgan** [comment]
I was able to replicate. Looks like the only easy fix is add one more way to define routes (in order). See #150.

This will be released asap to both divshot-cli and hosting. Most likely today!

**Comment by ice8lue** [comment]
Ah, so it really is a overwriting problem?

Okay, so I'll update and try the new method tomorrow. Thank you very much!

**Comment by scottcorgan** [comment]
@ice8lue the fix has been published both to `divshot-cli` and hosting.

See #150 notes for proper syntax to ensure the ordering of your routes.

Thanks for finding this!

**Comment by scottcorgan** [comment]
This is shipped also for Superstatic as version `2.1.0`
